lib/casmutils/xtal/coordinate.cxx

```diff
@@ -7,70 +7,39 @@ namespace casmutils
 {
 namespace xtal
 {
-Coordinate Coordinate::from_fractional(const Eigen::Vector3d& frac_coord, const Lattice& lat)
+namespace coordinate
 {
-    CASM::xtal::Coordinate coord(frac_coord, lat.__get(), CASM::FRAC);
-    return Coordinate(coord);
-}
-
-Coordinate Coordinate::from_fractional(double x, double y, double z, const Lattice& lat)
-{
-    return Coordinate::from_fractional(Eigen::Vector3d(x, y, z), lat);
-}
 
-void Coordinate::bring_within(const Lattice& lat)
+Eigen::Vector3d fractional_to_cartesian(const Eigen::Vector3d& fractional_coordinates, const Lattice& lat)
 {
-    this->casm_coord.set_lattice(lat.__get(), CASM::CART);
-    this->casm_coord.within();
-    return;
+    return CASM::xtal::Coordinate(fractional_coordinates, lat.__get(), CASM::FRAC).cart();
```

Comment:
I'm a little confused here because Coordinate is still being used. Does Coordinate still exist? If so, where?

Reply:
Oh nevermind, I see it's CASM. Whoops

Reply:
This is Coordinate in CASM. All the functions use CASM implementation.